editor/editor_settings.cpp

```diff
@@ -529,7 +529,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	set("text_editor/highlighting/highlight_all_occurrences", true);
 	set("text_editor/cursor/scroll_past_end_of_file", false);
 
-	set("text_editor/indent/type", 0);
+	set("text_editor/indent/type", "Tabs");
```

Comment:
I wouldn't if it wouldn't be better to define the enum as `Variant::INT` with two proper enum constants? @Paulb23